initfiles/componentfiles/configxml/daliplugin.xsd

```diff
@@ -0,0 +1,117 @@
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+################################################################################
+-->
+
+<?xml version="1.0" encoding="UTF-8"?>
+
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified" attributeFormDefault="unqualified">
+    <xs:element name="DaliServerPlugin">
+        <xs:annotation>
+            <xs:documentation>Describes a Dali Server Plugin</xs:documentation>
+        </xs:annotation>
+        <xs:complexType>
+            <xs:attribute name="build" type="buildType" use="required">
+              <xs:annotation>
+                <xs:appinfo>
+                  <tooltip>The build name to be deployed</tooltip>
+                  <viewType>hidden</viewType>
+                </xs:appinfo>
+              </xs:annotation>
+            </xs:attribute>
+            <xs:attribute name="buildSet" type="buildSetType" use="required">
+              <xs:annotation>
+                <xs:appinfo>
+                  <viewType>hidden</viewType>
+                </xs:appinfo>
+              </xs:annotation>
+            </xs:attribute>
+            <xs:attribute name="name" type="xs:string" use="optional">
+              <xs:annotation>
+                <xs:appinfo>
+                  <tooltip>Name for this dali server plugin</tooltip>
+                  <required>true</required>
+                </xs:appinfo>
+              </xs:annotation>
+            </xs:attribute>
+            <xs:attribute name="entrypoint" type="xs:string" use="optional" default="createWorkUnitFactory">
+              <xs:annotation>
+                <xs:appinfo>
+                  <tooltip>Name for this dali server plugin instance</tooltip>
+                  <required>true</required>
+                </xs:appinfo>
+              </xs:annotation>
+            </xs:attribute>
+            <xs:attribute name="daliServers" type="daliServersType" use="required">
+                <xs:annotation>
+                    <xs:appinfo>
+                        <tooltip>Specifies the dali server to which this plugin is associated with.</tooltip>
+                    </xs:appinfo>
+                </xs:annotation>
+            </xs:attribute>
+            <xs:attribute name="pluginName" type="xs:string" use="required">
+              <xs:annotation>
+                <xs:appinfo>
+                  <tooltip>name of the plugin</tooltip>
+                </xs:appinfo>
+              </xs:annotation>
+            </xs:attribute>
+            <xs:attribute name="type" type="xs:string" use="optional">
+              <xs:annotation>
+                <xs:appinfo>
+                  <tooltip>Specifies the type of the plugin</tooltip>
```

Comment:
can this be anything other than daliplugin? if so, do we have a list of supported types?